src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -240,103 +239,6 @@ public function hasAuthenticatedBackendUser()
 
 
 	/**
-	 * Add the template output to the cache and add the cache headers
-	 */
-	protected function addToCache()
```

Comment:
Okay, but it's not a use case we have to consider. If somebody uses the mobile layout, the SF reverse proxy has to consider this but the hook is not relevant.